src/ncp/ncp_buffer.hpp

```diff
@@ -354,29 +457,43 @@ class NcpFrameBuffer
         kSegmentHeaderNoFlag               = 0,          // No flags are set.
         kSegmentHeaderNewFrameFlag         = (1 << 15),  // Indicates that this segment starts a new frame.
         kSegmentHeaderMessageIndicatorFlag = (1 << 14),  // Indicates this segment ends with a Message.
+
+        kNumPrios                          = 2,          // Number of priorities.
```

Comment:
Given your last response, I think this should be; kNumPrios = kPriorityHigh + 1. Or something that connects kNumPrios with the enum it represents.